project.clj

```diff
@@ -1,33 +1,43 @@
-(defproject one-time "0.7.0"
+(defproject one-time "0.7.1"
   :author "Suvash Thapaliya"
   :description "One Time Password (TOTP and HOTP) library for Clojure. TOTP/HOTP is widely used for Two factor / Multi Factor Authentication."
   :url "http://github.com/suvash/one-time"
   :license {:name "Eclipse Public License"
             :url "http://www.eclipse.org/legal/epl-v10.html"}
 
-  :dependencies [[org.clojure/clojure "1.8.0"]
-                 [commons-codec "1.14"]
-                 [ring/ring-codec "1.1.2"]
-                 [com.github.kenglxn.qrgen/javase "2.6.0"]]
-  :profiles {:dev {:plugins      [[lein-cloverage "1.1.2"]
-                                  [lein-codox "0.10.7"]
-                                  [lein-ancient "0.6.15"]]}
+  :dependencies [[org.clojure/clojure "1.11.1"]
```

Comment:
I still want this package to be able to run on older (but lts supported) versions. This would be a major breakage. Since Clojure currently targets minimum supported java version to be 8, and 1.10.0 is the first one supporting that, I'll have this bumped only to 1.10.0